### ISSUE TITLE
Add second wave of jobs

### ISSUE THREAD

**Issue by saylorsd** [UCSUR-Pitt/wprdc-etl#52]
- [ ] Property Sales
- [ ] Assessment
- [ ] Crashes
- [ ] Dogs 
- [ ] Asbestos Permits

**Comment by bsmithgall** [comment]
@saylorsd I think that for this you might want to consider breaking this into two distinct projects.
1. The pipeline-generation code, which should have a cool name and be published to pip (happy to help out with this!)
2. The code for the actual pipelines.

**Comment by saylorsd** [comment]
@bsmithgall That sounds perfect!  What do you think would be the best way to go about it?  I was thinking I'll just make a branch where I remove the jobs and their tests and then push that to a new repo.

**Comment by saylorsd** [comment]
@bsmithgall Also... any ideas on a name?

**Comment by bsmithgall** [comment]
I'd recommend doing something like this: https://help.github.com/articles/duplicating-a-repository/

For names, I have no idea. Something about pipes?

**Comment by saylorsd** [comment]
@bsmithgall Since were going to keep the history anyway, would it just make sense to remove the jobs from this repo and rename it when we come up with something?

**Comment by saylorsd** [comment]
@bsmithgall I'm planning on removing the WPRDC-specific jobs and there tests and their respective git histories a la [this method](http://stackoverflow.com/a/17824718). 

I copied the jobs to [this repo](https://github.com/WPRDC/etl-jobs) with only their histories intact.  I think I'll move the current job tests and their respective mock files to a test directory within jobs and setup a separate build checks for them.

Does this make sense? 

 I think this way, when developing jobs, I'll simply add `wprdc-etl` as a requirement for pip and refactor the tests and stuff to account for that if necessary.

**Comment by bsmithgall** [comment]
Yeah. Ultimately we'll need to get the other thing installed up to [pypi](https://pypi.python.org/pypi). I am out on vacation next week, but the week after that I can probably dedicate some cycles.

**Comment by saylorsd** [comment]
@bsmithgall Sounds good! That'll give us some time to come up with a name for it. Have a good time!
